beaker-gke.gemspec

```diff
@@ -19,9 +19,16 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |file| File.basename(file) }
   s.require_paths = ['lib']
 
-  # required ruby version
+  # Required ruby version
   s.required_ruby_version = '~> 2.3'
 
+  # Testing dependencies
+  s.add_development_dependency 'fakefs', '~> 0.6', '<= 0.13.3'
+  s.add_development_dependency 'rake', '~> 10.1'
+  s.add_development_dependency 'rspec', '~> 3.0'
+  s.add_development_dependency 'rspec-its'
```

Comment:
I think we can drop this, we're not gonna write tests with `its`

Reply:
I don't think we can. This seems to be required by spec_helper of Beaker.

> An error occurred while loading spec_helper. - Did you mean?
    rspec ./spec/spec_helper.rb
Failure/Error: require File.join(beaker_spec_path, 'spec_helper.rb')
LoadError:
  cannot load such file -- rspec/its
.rvm/gems/ruby-2.6.3/gems/beaker-4.13.1/spec/spec_helper.rb:10:in `require'
.rvm/gems/ruby-2.6.3/gems/beaker-4.13.1/spec/spec_helper.rb:10:in `<top (required)>'
./spec/spec_helper.rb:14:in `require'
./spec/spec_helper.rb:14:in `<top (required)>'
No examples found.